docs/modular-resources/configure.md

```diff
@@ -133,6 +134,91 @@ For any version type other than **Patch (X.Y.Z)**, the module will upgrade as so
 If, for example, the module provides a motor component, and the motor is running, it will stop while the module upgrades.
 {{% /alert %}}
 
+### Use environment variables with a registry module
+
+Some modules require that you set specific environment variables as part of configuration.
+You can click the **URL** link in the upper-right corner of the module configuration pane to view any specific requirements on the module's GitHub page.
+
+You can configure environment variables for a module in two ways:
+
+- As a static value that does not change during runtime.
+- As a reference to a system environment variable that can be changed during runtime.
+  Any system environment variable available to the `viam-server` instance can be referenced in this way.
+
+Using static values is suitable for instances where a configuration does not need to change.
+Referencing environment variables allows you to change the value of that variable at any point, effectively changing the behavior of the modular resource, without needing to restart `viam-server` for the change to take effect.
+For example, you could configure an environment variable to control the hue tolerance for a [color detector](/services/vision/detection/), and decide to change that value later based on the ambient brightness of the sun.
```

Comment:
I think there is a bit of confusion here.

Environment variables are all just strings, however, the values in these strings will be replaced using the standard viam-server placeholder replacement logic (ex: ${packages.ml_model.foo} gets replaced with the location of foo on the computer, and ${environment.PATH} gets replaced with viam-servers path variable. This is useful if you want to add something to the PATH ex: "PATH":"~/go/bin:${environment.PATH}" (which is the standard way to append environment variables))